install/k8s/install.sh

```diff
@@ -229,15 +229,14 @@ else
 fi
 aci_key=./aci.key
 
-set +e
-$kubectl get secret aci.key -n kube-system &>/dev/null
-set -e
+aci_key_check_code=0
+$kubectl get secret aci.key -n kube-system &>/dev/null || aci_key_check_code=$?
 
```

Comment:
not specific to this commit,
 ```echo``` commands are redundant here.
```$kubectl create ``` already logs message to console.
we can modify to ``` $kubectl get .... || kubectl create ...```